pkgs/development/python-modules/ocrmypdf/default.nix

```diff
@@ -0,0 +1,118 @@
+{ buildPythonPackage
+, cffi
+, chardet
+, fetchFromGitHub
+, ghostscript
+, hocr-tools
+, img2pdf
+, isPy3k
+, jbig2enc
+, leptonica
+, pdfminer
+, pillow
+, pikepdf
+, pngquant
+, pypdf2
+, pytest
+, pytest-helpers-namespace
+, pytest_xdist
+, pytestcov
+, pytestrunner
+, python-xmp-toolkit
+, qpdf
+, reportlab
+, ruffus
+, setuptools
+, setuptools-git
+, setuptools_scm
+, setuptools-scm-git-archive
+, stdenv
+, tesseract4
+, unpaper
+}:
+
+buildPythonPackage rec {
+  pname = "ocrmypdf";
+  version = "8.2.3";
+  disabled = ! isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "jbarlow83";
+    repo = "OCRmyPDF";
+    rev = "v${version}";
+    sha256 = "1ldlyhxkav34y9d7g2kx3d4p26c2b82vnwi0ywnfynb16sav36d5";
+  };
+
+  buildInputs = [
+    pytest
+    pytestrunner
+    setuptools
+    setuptools_scm
+    setuptools-scm-git-archive
+  ];
+
+  propagatedBuildInputs = [
+    cffi
+    chardet
+    img2pdf
+    pdfminer
+    pikepdf
+    reportlab
+    ruffus
+  ] ++ runtimeDeps;
+
+  checkInputs = [
+    hocr-tools
+    pypdf2
+    pytest
+    pytest-helpers-namespace
+    pytest_xdist
+    pytestcov
+    pytestrunner
+    python-xmp-toolkit
+    setuptools
+  ] ++ runtimeDeps;
+
+  runtimeDeps = [
```

Comment:
This is not a valid attribute and should be moved to a `let ... in` expression.

Reply:
How are these used? From `PATH`? Is this a library or a cli tool? If it's the latter, then the entry point will be wrapped and these tools will be on `PATH`, which i s fine. However, if it's also to be used as a library, then these dependencies cannot be found. Preferably the references to them are patched.